raiden-ts/tests/unit/epics/channels.spec.ts

```diff
@@ -408,7 +408,9 @@ describe('channelDepositEpic', () => {
       channelDeposit.failure(expect.any(Error), { tokenNetwork, partner: partner.address }),
     );
     expect(tokenContract.functions.approve).toHaveBeenCalledTimes(1);
-    expect(tokenContract.functions.approve).toHaveBeenCalledWith(tokenNetwork, deposit);
+    expect(tokenContract.functions.approve).toHaveBeenCalledWith(tokenNetwork, deposit, {
```

Comment:
Actually there is a case for the manual nonce missing. :stuck_out_tongue_winking_eye: